src/main/java/uk/co/binaryoverload/hugapi/Hug.java

```diff
@@ -106,6 +106,7 @@ public Hug create() {
         MUM_HUG,
         COMFORTING,
         BRITISH,
+        PHATT,
```

Comment:
What in mother of mary is this... xD

Reply:
A really squishy warm hug, slang in my secondary school is ott

Reply:
Find something else that people can understand 👀

Reply:
okey dokey